examples/medplum-patient-intake-demo/src/pages/UploadDataPage.tsx

```diff
@@ -72,10 +66,130 @@ export function UploadDataPage(): JSX.Element {
   );
 }
 
-async function uploadData(medplum: MedplumClient, data: Bundle): Promise<void> {
-  const result = await medplum.executeBatch(data);
+async function uploadCoreData(medplum: MedplumClient): Promise<void> {
+  const batch = coreData as Bundle;
+
+  const result = await medplum.executeBatch(batch);
+
+  showNotification({
+    icon: <IconCircleCheck />,
+    title: 'Success',
+    message: 'Uploaded Business Statuses',
```

Comment:
"Business Status" is not the right description here. We need to update this message

Reply:
Why do we need this notificaiton, as we have another success notification down below